plugins/pulp_docker/plugins/db/models.py

```diff
@@ -0,0 +1,19 @@
+from mongoengine import (IntField, StringField)
+
+from pulp.server.db.model import ContentUnit
+
+
+class DockerImage(ContentUnit):
+    image_id = StringField(required=True)
+    parent_id = StringField()
+    size = IntField(required=True)
```

Comment:
See the comment on the original model class for the "size" parameter, where it explains why this can be ``None``. Have you taken that into account somehow?